plugins/kubernetes-backend/src/service/KubernetesBuilder.ts

```diff
@@ -239,6 +242,38 @@ export class KubernetesBuilder {
       }
     });
 
+    router.post('/resources/workloads', async (req, res) => {
```

Comment:
Thoughts on the alternative suggested in https://github.com/backstage/backstage/issues/11309#issuecomment-1117483832?

Reply:
I didn't have time to look into that yesterday but I think it makes sense in principle

Reply:
done